### ISSUE TITLE
Error inflating class android.support.v7.widget.CardView

### ISSUE THREAD

**Issue by Hesamedin** [robolectric/robolectric#1920]
Hi would you please have a look at my issue on stackoverflow please. Thanks.
http://stackoverflow.com/questions/31175076/how-to-test-an-activity-with-robolectric-while-it-has-android-support-v7-widget

**Comment by jaredsburrows** [comment]
Please read this before posting issues: https://github.com/robolectric/robolectric/wiki/How-to-Report-Issues.

I have tried debugging your issue on StackOverflow last night. I need more information.

Run those tests with `gradlew testDebug --info`.

Many of my applications use `RecyclerView + CadView`, I can also click on items. Can you should us code?

**Comment by Hesamedin** [comment]
@jaredsburrows Thanks bro for your fast reply and appreciate for the time you spent on this issue.
I just created a new project (since I cannot share the project of company) in order to replicate the issue. However, I'm happy to say that, I could get it to work/test successfully. 
As I mentioned I'm using just CardView in my splash screen.

I have pushed my code to the following repo for those who have same problem.
https://github.com/Hesamedin/RobolectricTester/tree/master/app/src/test
